src/Components/Donate/Donate.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import './Donate.css'
+import DonateBox from "./DonateBox";
+import DonateDescription from "./DonateDescriptoin";
+
+
+const whyDonate = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.";
```

Comment:
- [ ] Update content

Reply:
I have requested the content. Will update.